static-sites.yml

```diff
@@ -24,7 +24,7 @@ all:
   cinescuela.es:
     name: Cinescuela
     version: 2016-10-20
-    description: Lleva el cine a tu aula de clase.
+    description: Lleva el cine a tu biblioteca móvil.
```

Comment:
All ideasbox are "biblioteca movil" at the end, and we may say the same for koombook I think. ;)
Plus, we cannot say that we are more in a "aula de clase" (classroom) than in a biblbioteca.